root/area/edit_form.tt

```diff
@@ -42,6 +42,16 @@
 
 <script type="text/javascript">//<![CDATA[
   (function () {
+    const wasEnded = $('#id-edit-area\\\.period\\\.ended').prop('checked');
```

Comment:
Or if you'd prefer, you can wrap the code in a function which accepts the entity type as a parameter, then put the function on the global `MB` object, and call it here.